wikiman.sh

```diff
@@ -285,6 +296,23 @@ while getopts l:s:H:pqhRSk o; do
 done
 shift "$((OPTIND - 1))"
 
+# Dependency check
+
+echo 'Checking dependencies...'
+for dependency in 'man' 'fzf' 'rg' 'awk' 'w3m' "$conf_fuzzy_finder"; do
+	which "$dependency" >/dev/null || {
+		echo "Error: missing dependency: cannot find $dependency executable!"
+		exit 1
+	}
+done
+
+# Check if fuzzy finder compatible with used fzf's parameters
+
+[ "$conf_fuzzy_finder" = "fzf" ] || [ "$conf_fuzzy_finder" = "sk" ] || {
+	echo "Error: $conf_fuzzy_finder is not compatible with the paramters used in this script"
+	exit 1
+}
```

Comment:
- This block shoud use `if-then-fi` syntax to follow the style.
- Print error to *STDERR*.
- Every fatal error has its own code.
- Single quotes for regular strings
```bash
if [ "$conf_fuzzy_finder" = 'fzf' ] || [ "$conf_fuzzy_finder" = 'sk' ]; then
	echo "error: $conf_fuzzy_finder is not compatible with the paramters used in this script" 1>&2
	exit 4
fi
```

Reply:
> This block shoud use `if-then-fi` syntax to follow the style.

The check would not work with the `if-then-fi` syntax, so I change it to `case-esac` syntax instead.